src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,79 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        FileInputStream fis = null;
+        try {
+            fis = new FileInputStream(fromFileName);
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Can't read data from the file", e);
+        }
+        byte[] buffer = new byte[10];
+        StringBuilder sb = new StringBuilder();
+        while (true) {
+            try {
+                if (fis.read(buffer) == -1) {
+                    break;
+                }
+            } catch (IOException e) {
+                throw new RuntimeException("Can't read data from the file", e);
+            }
+            sb.append(new String(buffer));
+            buffer = new byte[10];
+        }
+        try {
+            fis.close();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from the file", e);
+        }
+        String content = sb.toString();
+        String nw = content.replaceAll("[.!/?*',]", " ").toLowerCase();
+        String n = nw.replaceAll("[\n\r]", " ");
```

Comment:
Your code replaces various punctuation marks and converts the text to lowercase. However, this approach may not handle all possible cases correctly. Also please choose the better names for you variables. There are not the best - `n`, `nw`.